src/components/comment/Comment.tsx

```diff
@@ -3,6 +3,8 @@ import CommentInfo from '../commentInfo/CommentInfo';
 import ProfilePicture from '../profilePicture/ProfilePicture';
 import CommentContent from '../commentContent/CommentContent';
 import LikeCommentButton from '../likeCommentButoon/LikeCommentButton';
+import { classes as generalClasses } from '../../style/general.st.css';
+import { classes } from './Comment.st.css';
 import type { Comment as CommentType } from '../../types';
 
 interface IComment extends CommentType {}
```

Comment:
whats this?

Reply:
Make all the componnents with the same interface naming format => I + component's name.
Here type and component has the same name